src/sentry/tagstore/multi/backend.py

```diff
@@ -1,288 +0,0 @@
-"""
-sentry.tagstore.multi.backend
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-:copyright: (c) 2010-2017 by the Sentry Team, see AUTHORS for more details.
-:license: BSD, see LICENSE for more details.
-"""
-
-from __future__ import absolute_import
-
-import logging
-import six
-import random
-from threading import Thread
-from six.moves.queue import Queue, Full
-from operator import itemgetter
-
-from sentry import options
-from sentry.tagstore.base import TagStorage
-from sentry.utils import metrics
-from sentry.utils.imports import import_string
-
-
-logger = logging.getLogger('sentry.tagstore.multi')
-
-
-class QueuedRunner(object):
-    """\
-    Secondary backend runner that puts method calls on a bounded queue and drops them
-    when the queue is full.
-
-    A separate (non-main) thread works the queue.
-    """
-
-    def __init__(self):
-        self.q = Queue(maxsize=100)
```

Comment:
Which functionality specifically? The bounded queue?

Reply:
Yeah.

Reply:
It can be passed through as executor options:

https://github.com/getsentry/sentry/blob/57c1fbaa0a51397aba45b656db9489397eb8b3f5/src/sentry/utils/services.py#L110-L115

https://github.com/getsentry/sentry/blob/57c1fbaa0a51397aba45b656db9489397eb8b3f5/src/sentry/utils/concurrent.py#L133-L136

Reply:
:sweat_smile: